README.md

```diff
@@ -5,20 +5,30 @@ eLife 2.0 Dummy API
 
 This contains a dummy implementation of the [eLife 2.0 API](https://github.com/elifesciences/api-raml).
 
-##Import article
+## Import article
 
 ```$sh
-cd /srv/api-dummy
-./bin/import 09560
+docker compose run app ./bin/import 09560
```

Comment:
This is the only task I run into problems with, perhaps a Linux file ownership issue:
```
[16:23:06][giorgio@Arconia:~/code/api-dummy]$ docker compose run app ./bin/import 92110
WARN[0000] /home/giorgio/code/api-dummy/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion 
WARN[0000] Found orphan containers ([api-dummy-app-run-6f44d91c5293 api-dummy-composer-run-1984308df2a2 api-dummy-composer-run-73e461503125]) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up. 
[+] Creating 1/0
 ✔ Container api-dummy-composer-1  Created                                                                                                        0.0s 
[+] Running 1/1
 ✔ Container api-dummy-composer-1  Started                                                                                                        0.2s 
PHP Fatal error:  Uncaught Symfony\Component\Filesystem\Exception\IOException: Failed to touch "/srv/api-dummy/bin/../data/articles/92110.json". in /srv/api-dummy/vendor/symfony/filesystem/Filesystem.php:150
Stack trace:
#0 /srv/api-dummy/bin/import(66): Symfony\Component\Filesystem\Filesystem->touch('/srv/api-dummy/...')
#1 {main}
  thrown in /srv/api-dummy/vendor/symfony/filesystem/Filesystem.php on line 150
```

Reply:
`./bin/import 09560` remains available as a workaround.